conda/common/logic.py

```diff
@@ -910,19 +911,20 @@ def minimal_unsatisfiable_subset(clauses, sat, explicit_specs):
     <= sat(A)), where A <= B means A is a subset of B and False < True).
 
     """
-    working_set = set(explicit_specs)
+    working_set = set()
     found_conflicts = set()
 
+    if sat(explicit_specs, True) is None:
```

Comment:
This is a smart way to trim the number of SAT solves when there's no conflict.  Good thinking.